backend/cfg/cfg_with_layout.ml

```diff
@@ -64,6 +64,16 @@ let remove_block t label =
   t.layout <- List.filter (fun l -> not (Label.equal l label)) t.layout;
   t.new_labels <- Label.Set.remove label t.new_labels
 
+let remove_blocks t labels_to_remove =
```

Comment:
I think we should leave a comment
somewhere that the type of `layout`
is inadequate, and that bulk removal
is a temporary workaround.